### PR TITLE
Add Responsivness to the portfolio Desktop version

### PR DESCRIPTION
**Pull request by Hamdullah-Andar**
## Desktop version Complete Portfolio

- Update CSS
- Add Flex-box
- Add Grid
- Add Media Query

## Thank you for the review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,13 +18,18 @@
...
                 </nav>
             </header>
 
             
            <!-- The main section  -->
-            <main>
+            <main class="main-class">
                 <section class="hero">
 
                 <h1 class="heading">I’m Hami
```
Comment (ha-manel) [inline index.html]: - [x] Kindly update your headline section for desktop version to make it look as much as possible like Figma. The `width` and text `color` are not the same.

## Your design:
image

## Figma design:
image

File: index.html [hunk 2]
```diff
@@ -18,13 +18,18 @@
...
+                        <a class="about-menu" href="">About</a>
+                        <a class="contact-menu" href="">Contact</a>
+                    </div>
                 </nav>
             </header>
 
             
            <!-- The main section  -->
-            <main>
+            <main class="main-class">
```
Comment (ha-manel) [inline index.html:32]: - [x] Kindly remove the empty space on the right side of the website 👍 
Animation

File: index.html [hunk 3]
```diff
@@ -55,10 +60,13 @@ <h3 class="connect">LET’S CONNECT</h3>
...
                     <div class="section1">
-    
+                        <!-- <img src="images/Snapshoot-Portfolio-Disk1.png" alt=""> -->
                     </div>
+                </section>
+                
+                <section class="project-description">
                     <div>
                         <h2>Tonic</h2>
                         <ul class="tonic-title">
```
Comment (ha-manel) [inline index.html:72]: - [x] Kindly follow the Figma design as much as possible, this includes `font-size`, text `color`, and the `background-color` of the technologies badges 👍 

## Your design:
image

## Figma design:
image

File: index.html [hunk 4]
```diff
@@ -81,19 +89,22 @@ <h2>Tonic</h2>
                     <div>
                         <button class="project-button">See Project</button>
```
Comment (ha-manel) [inline index.html:90]: - [x] Kindly add button interaction styles (hover, clicked...) as shown in the Figma design:

image

File: index.html [hunk 5]
```diff
@@ -179,12 +196,13 @@ <h2>Multi-Post Stories</h2>
...
                 </div>
             </section>
 
             <section class="about">
                 <div class="about-container">
-
+                <section class="about-headings">    
                     <h2 class="about-heading">About
                         Myself
                     </h2>
```
Comment (ha-manel) [inline index.html]: - [x] Kindly follow the same `font-size` and text `color` as shown on Figma, as well as button interaction styles 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Hamdullah-Andar 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [x] The website should look good in all screen sizes, kindly fix the issue that happens in certain screen widths to make your project look awesome across all device sizes 💯 

Animation


_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — APPROVED** [review]
Hi @Hamdullah-Andar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**